release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3127,6 +3127,8 @@ In the following tables, features are marked with the following statuses:
 
 * When using CPU load balancing on cgroupv2, a pod can fail to start if another pod that has access to exclusive CPUs already exists. This can happen when a pod is deleted and another one is quickly created to replace it. As a workaround, ensure that the old pod is fully terminated before attempting to create the new one. (link:https://issues.redhat.com/browse/OCPBUGS-34812[*OCPBUGS-34812*]) 
 
+* Enabling LUKS encryption on a system using 512 emulation disks causes provisioning to fail and the system launches the emergency shell in the initramfs. This happens because of an alignment bug in sfdisk when growing a partition. As a workaround, the provisioning step `ignition-ostree-growfs` can now detect this issue, and the system can continue provisioning. (link:https://issues.redhat.com/browse/OCPBUGS-35410[*OCPBUGS-35410*])
```

Comment:
Hi @jldohmann, the part where you say "*the provisioning step `ignition-ostree-growfs` can now detect this issue*" is precisely what @jlebon fixed with the patch, not the workaround itself, maybe we can use something like this?:
~~~
Enabling LUKS encryption on a system using 512 emulation disks causes provisioning to fail and the system launches the emergency shell in the initramfs. This happens because of an alignment bug in `sfdisk` when growing a partition, a workaround for this is to have ignition do the resizing instead (link:https://issues.redhat.com/browse/OCPBUGS-35410?focusedId=24924697&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24924697)
~~~

Reply:
Sorry, saw this after posting my suggestion in the other comment. OK, how about I duplicate my comment in https://github.com/coreos/fedora-coreos-tracker/issues/1384#issuecomment-2189239351 in the Jira issue and we can link to that?

Reply:
we typically just link to the whole ticket so that the customer has all the context needed for the issue. can the workaround in the ticket itself be pinned for easy access?

Reply:
Thanks, pinned!